ddtrace/monkey.py

```diff
@@ -25,7 +25,7 @@
     'cassandra': True,
     'celery': True,
     'elasticsearch': True,
-    'futures': False,  # experimental propagation
+    'futures': True,
```

Comment:
This integration is actually well tested only on Tornado environments (and Tornado enables it by default). For other scenarios, I think it must be disabled until we have more feedbacks from users.

For now it's better to keep it disabled, while everything else is enabled.